rfcs/20191017-tfx-standardized-inputs.md

```diff
@@ -116,6 +134,47 @@ to compute approximate heavy hitters over this in-memory representation. We can
 now share this implementation inside both TFDV and TFT for their top-K feature
 value computation.
 
+# Requirements
+
+The in-memory representation should:
+
+*   Be columnar.
+
+    A columnar in-memory representation works better than a row based on under
+    typical workload of TFX libraries:
+    +   Compute statistics over a column.
+    +   Feed a batch of rows to TensorFlow.
+
+*   Be able to losslessly encode the logical data format. Specifically, it
+    should be able to distinguish a null (unpopulated) value from an empty list.
```

Comment:
What is an empty list in this context? Can you give an example of when the two distinct values appear?